pkg/reconciler/apis/apibinding/apibinding_reconcile.go

```diff
@@ -318,6 +317,7 @@ func (c *controller) reconcileBinding(ctx context.Context, apiBinding *apisv1alp
 			c.deletedCRDTracker.Remove(crd.Name)
 
 			needToWaitForRequeueWhenEstablished = append(needToWaitForRequeueWhenEstablished, schemaName)
+			continue
```

Comment:
okay, I think I understand, skip it until it becomes `established`

Reply:
a CRD is marked as established by the `api-extensions` server

Reply:
yes

Reply:
We should not show a resource as bound until that has happened.